test/mock_data_stream.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import pytest
+
+
+class MockStreamData:
+    def __init__(
+        self,
+        *,
+        stream_id=1,
+        srate=None,
+        tdiff=None,
+        fmt="float64",
+        time_stamps=[],
+        time_series=None,
+        clock_times=[],
+        clock_values=[],
+    ):
```

Comment:
Could you add a docstring and describe all parameters here? IMO these are not all immediately obvious (especially `tdiff`).

Reply:
Done!